map/src/manager/track/SaveTrackManager.js

```diff
@@ -374,7 +373,7 @@ function getOldSizeTrack(currentTrackIndex) {
 
 // after success upload from Local to Cloud
 // download it and use as current Cloud track
-async function downloadAfterUpload(ctx, file) {
+async function downloadAfterUpload(ctx, file, trackData) {
```

Comment:
Actually, trackData is not fully used, so possibly better to change it to `showOnMap` parameter with default option specified.